L1Trigger/L1TGlobal/interface/L1TGlobalUtilHelper.h

```diff
@@ -131,8 +135,16 @@ namespace l1t {
     // Register the callback function with the Framework
     // if any InputTags still need to be found.
     if (findL1TAlgBlk || findL1TExtBlk) {
-      module.callWhenNewProductsRegistered([this, findL1TAlgBlk, findL1TExtBlk, iC](auto iBranch) {
-        checkToUpdateTags(iBranch, iC, findL1TAlgBlk, findL1TExtBlk);
+      assert(false);
```

Comment:
what's the point of the code after `assert(false)` ?

Reply:
oops. Left over from when I was running in the debugger. Thanks for the catch.